src/github.rs

```diff
@@ -60,7 +60,7 @@ impl Notification {
     /// This will return an error if the subscription's subject is not a pull request.
     pub fn pull_request(&self) -> Result<PullRequest, Box<dyn Error>> {
         if self.subject.r#type != SubjectType::PullRequest {
-            return Err("subscription is not a pull request")?;
+            return Err("subscription is not a pull request".into());
```

Comment:
Is that an edition idiom warning? Seems surprising to me.

Reply:
I'm not sure, just clippy warned about this.